jib-build-plan/build.gradle

```diff
@@ -0,0 +1,45 @@
+plugins {
+  id 'net.researchgate.release'
+  id 'maven-publish'
+  id 'eclipse'
+}
+
+dependencies {
+  implementation "com.google.guava:guava:${dependencyVersions.GUAVA}"
```

Comment:
It makes sense that we need this, but I would think a library like this should almost have zero deps to minimize diamond dep issues.

Reply:
We'll need to do some more work without this, but I agree. I'll move this to `testImplementation`.